crates/cxx-qt-gen/test_outputs/properties.h

```diff
@@ -26,10 +26,11 @@ class MyObject : public QObject
 
 public:
   ~MyObject();
+
+public:
   MyObjectRust const& unsafeRust() const;
   MyObjectRust& unsafeRustMut();
 
-public:
```

Comment:
Why do these need to become public?

Reply:
They were public before ...

```cpp
public:
  ~MyObject();

  MyObjectRust const& unsafeRust() const;
  MyObjectRust& unsafeRustMut();

public:
  ::std::int32_t const& getPrimitive() const;
  Q_SLOT void setPrimitive(::std::int32_t const& value);
...
```

... and they are public after ?

```cpp
public:
  ~MyObject();

public:
  MyObjectRust const& unsafeRust() const;
  MyObjectRust& unsafeRustMut();

  ::std::int32_t const& getPrimitive() const;
  Q_SLOT void setPrimitive(::std::int32_t const& value);
```

Just the `unsafeRust` and `unsafeRustMut` are now generated in the same `public:` block as the `getPrimitive` whereas before they were separate.

Reply:
ah, I didn't see that they were public before